content/collections/experiment/en/dimensional-analysis.md

```diff
@@ -0,0 +1,22 @@
+---
+id: f544b5a1-0689-4137-ab14-690903ae7902
+blueprint: experiment
+title: 'Dimensional Analysis'
+landing: false
+exclude_from_sitemap: false
+updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+updated_at: 1737480058
+---
+
+Sometimes, you might want to remove QA users or other internal traffic from your analyses because they're not representative of your customer base, and may skew results.
+
+Amplitude's Dimensional Analysis capabilities enable you to exclude groups of users that you define from analysis on a per-experiment basis.
+
+## Define your testers
```

Comment:
This section header is about "testers" but there is information that is not specific to "testers"

Reply:
We should probably retire this article and move the information about testers to somewhere more logical.

Reply:
this is a new article right?